__init__.py

```diff
@@ -1,3 +1,3 @@
-from loader import releasenotes
+from thunderbird_notes.loader import releasenotes
```

Comment:
Python 3 won't do file-relative imports, only package-relative.  Python 2.7 will do it either way, so the changed reference still works in 2.7. This does make the assumption that it's in a directory named thunderbird_notes, though (which it will be if you followed the directions for the main website build checkout, but not if you checked it out standalone....)